src/gt4py/test_builder.py

```diff
@@ -0,0 +1,127 @@
+import sys
```

Comment:
Can the TestBuilder be a separate PR from the dawn integration part 2?

Reply:
Removed.